CMakeLists.txt

```diff
@@ -362,6 +362,7 @@ set(ZIG_STAGE2_SOURCES
     "${CMAKE_SOURCE_DIR}/lib/std/io/buffered_writer.zig"
     "${CMAKE_SOURCE_DIR}/lib/std/io/change_detection_stream.zig"
     "${CMAKE_SOURCE_DIR}/lib/std/io/counting_writer.zig"
+    "${CMAKE_SOURCE_DIR}/lib/std/io/counting_reader.zig"
```

Comment:
I don't think you need to update this unless you actually use it in the self hosted compiler.

Reply:
For now its a list of every zig stdlib file.
Otherwise someone needs to remember to update it when they use certain functions in stage2